examples/multisig_plain/lib.rs

```diff
@@ -255,8 +260,36 @@ mod multisig_plain {
         }
 
         /// Add a new owner to the contract.
-        /// Panics if the owner already exists.
+        ///
         /// Only callable by the wallet itself.
+        ///
+        /// # Panics
+        ///
+        /// If the owner already exists.
+        ///
+        /// # Examples
+        ///
+        /// Since this message must be send by the wallet itself it has to be build as a
+        /// `Transaction` and dispatched through `submit_transaction` + `invoke_transaction`:
+        /// ```ignore
+        /// use ink_core::env::call{CallData, Selector};
+        /// // first create the transaction that add `alice` through `add_owner`
+        /// let alice: AccountId = [1u8; 32].into();
+        /// let mut call = CallData::new(Selector::from_str("add_owner"));
+        /// call.push_arg(&alice);
+        /// let transaction = Transaction {
+        ///     callee: THIS_WALLET,
+        ///     selector: call.selector().to_bytes(),
+        ///     input: call.params().to_owned(),
+        ///     transferred_value: 0,
+        ///     gas_limit: 5000
+        /// };
+        ///
+        /// // submit the transaction for confirmation
+        /// let (id, _) = submit_transaction(transaction);
+        /// // wait until all required owners have confirmed and then execute the transaction
+        /// invoke_transaction(id);
+        /// ```
```

Comment:
nice

Reply:
I reworked it to use `CallParams` to call into the contract. So it's less pseudo code and will actually compile once rustdocs will try to compile the example.